app/vector_database/lecture_transcription_schema.py

```diff
@@ -0,0 +1,106 @@
+from enum import Enum
+
+from weaviate.classes.config import Property
+from weaviate import WeaviateClient
+from weaviate.collections import Collection
+from weaviate.collections.classes.config import Configure, VectorDistances, DataType
+
+
+class LectureTranscriptionSchema(Enum):
+    """
+    Schema for the lecture transcriptions
+    """
+
+    COLLECTION_NAME = "LectureTranscriptions"
+    COURSE_ID = "course_id"
+    COURSE_NAME = "course_name"
+    LECTURE_ID = "lecture_id"
+    LECTURE_NAME = "lecture_name"
+    LANGUAGE = "language"
+    SEGMENT_START = "segment_start"
+    SEGMENT_END = "segment_end"
```

Comment:
Is this the Segment start time? If so please rename the values:
```suggestion
    SEGMENT_START_TIME = "segment_start_time"
    SEGMENT_END_TIME = "segment_end_time"
```